src/databricks/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.9.0
++++++
+* az databricks workspace and az databricks workspace vnet-peering: Migrate to atomic commands

```

Comment:
"Command is atomic or not" is transparent to users, so we don't need to declare it in changelog.